distributed/protocol/tests/test_serialize.py

```diff
@@ -190,9 +190,19 @@ def test_empty_loads_deep():
     assert isinstance(e2[0][0][0], Empty)
 
 
-def test_serialize_bytes():
-    for x in [1, "abc", np.arange(5), b"ab" * int(40e6)]:
-        b = serialize_bytes(x)
+@pytest.mark.parametrize(
+    "kwargs", [{}, {"serializers": ["pickle"]},],
+)
+def test_serialize_bytes(kwargs):
+    for x in [
+        1,
+        "abc",
+        np.arange(5),
+        b"ab" * int(40e6),
+        2 ** 26 * b"ab",
+        (2 ** 25 * b"ab", 2 ** 25 * b"ab"),
```

Comment:
Beware here that I've seen Python pre-compute literals like these as an unfortunate optimization, which results in large constant values in the program code.  It might be wise to call a function around `2 **26` like `int(2 ** 26)` even though it has no semantic effect.

My bad experience on this was a long time ago, and so maybe things have improved..

Reply:
Hmm...yeah I'm not sure. Did push some `int` wrappers around these though. One of these cases was already here (so we may have already been ok).

Reply:
Ah ok, probably not a problem then.  Thanks again for resolving the full issue quickly.

Reply:
Of course 🙂